validator/sawtooth_validator/journal/block_validator.py

```diff
@@ -89,22 +89,28 @@ def merge_block_result(self, block_validation_result):
 class BlockValidationResult:
     def __init__(self, block):
         self.block = block
-        self.execution_results = []
-        self.transaction_count = 0
 
     def __bool__(self):
         return self.block.status == BlockStatus.Valid
 
+    @property
+    def execution_results(self):
+        return self.block.execution_results
+
+    @property
+    def transaction_count(self):
+        return self.block.execution_results
+
     def set_valid(self, is_valid):
         if is_valid:
             self.block.status = BlockStatus.Valid
         else:
             self.block.status = BlockStatus.Invalid
 
     def merge_batch_results(self, batch_validation_results):
-        self.transaction_count =\
+        self.block.transaction_count +=\
```

Comment:
Unrelated to your changes, can you add a space between `=` and `\`? I always look at `=\` and think it's doing division or something, even with the `+`.